### ISSUE TITLE
Feature Request: Ability To Save Outputs in the source file

### ISSUE THREAD

**Issue by jlewi** [stateful/runme#701]
It would be nice to have an option to save the output of the code cell in the source markdown. As a consequence of this, execution outputs would persist across kernel sessions. 

Right now the auto-save feature saves output in a separate markdown document timestamped with a ULID extension.
https://docs.runme.dev/usage/auto-save

This makes sense when a markdown document is intended to be a reusable document (e.g. a README) that gets run multiple times.

Its much less useful when the goal is to use Runme to perform a set of operations and produce a log of all the commands that was run. For example, if I'm performing an operational task; e.g. "spin up a new AKS cluster" I want to produce a single file that is a record of everything that happened before. Right now with the auto-save feature this often gets spread across multiple output markdown files because a new file is created each time the session is restarted. In the course of performing a task I will often create multiple sessions (e.g. as a side effect of saving/closing the file).

The other problem in this case with storing the output of previous cells in a separate file is that the output provides important context for the cells that come after it. So if the outputs get saved in a separate file the document is no longer readable. 

To work around this; I often find myself copying the output of code cells into a markdown cell to preserve the record in a readable way.

**Comment by sourishkrout** [comment]
Makes sense. One question, @jlewi, when you run a single cell multiple times, would you expect the only latest output to be preserved or all of the previous runs to be preserved?

I'm getting the sense that in this mode, the notebook should work a lot like a like a terminal session.
